Makefile

```diff
@@ -17,7 +17,7 @@ install:
 
 example: default
 	protoc -I. -I$(SRCPATH) -I./vendor \
-		--go_out="plugins=grpc:$(SRCPATH)" --gorm_out="$(SRCPATH)" \
+		--go_out="plugins=grpc:$(SRCPATH)" --gorm_out="engine=postgres:$(SRCPATH)" \
```

Comment:
if you don't add the `engine=postgres` part, does it default to postgres or does it crash?

Reply:
Neither in general. However because the marshal test assume the JSON field exists they'll fail to build.

Reply:
ah, interesting. I guess if we were trying to be pure, we'd have a proto run for each engine we know about (2 in this case) and tests or whatever for all of them